setup.py

```diff
@@ -1,12 +1,14 @@
 from distutils.core import setup
 
 setup(
-    name='CameraTestSetup',
+    name='cts',
```

Comment:
I think this renaming here is very good.

Reply:
I have seen some warning during cts installation which was probably caused by that. Hopefully it will fix it :-)